yaml/ruleset.go

```diff
@@ -137,12 +137,24 @@ func (r *Rules) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err == nil {
 		r.Branch = rules.Branch
 		r.Comment = rules.Comment
-		r.Event = rules.Event
 		r.Path = rules.Path
 		r.Repo = rules.Repo
 		r.Status = rules.Status
 		r.Tag = rules.Tag
 		r.Target = rules.Target
+
+		// account for users who use non-scoped pull_request event
+		events := []string{}
+
+		for _, e := range rules.Event {
+			if e == "pull_request" {
+				events = append(events, "pull_request:opened", "pull_request:synchronized")
```

Comment:
Couple of questions:

- Should you utilize the constants defined above?
- Why is `edited` not included?

Reply:
Yeah using the constants is probably a better idea.

As for the absence of `edited`, I wanted to make sure that the `pull_request` event wouldn't change behavior if it wasn't changed. Since we currently only react to `opened` and `synchronize` actions, adding `edited` would result in unexpected behavior. Although, the overall scoping pattern would be much better sensical if `pull_request` applied to all actions. Open for more discussion on that front

Reply:
I think we would want to do something like `pull_request:*` or `pull_request:all` to allow backwards compatibility.